Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "gzip"
+version = "2.0.0"
```

Comment:
nit: put alpha or beta or some other indicator that this is not a release. Or in more old-school style, 1.99.0.